### PR TITLE
Rework the end of the Install Sensu guide

### LINKED ISSUE #2555: Add more accessible pathway at the end of the Install Sensu doc
### Description
As summarized by Vikas: "We need to wrap up the Install Sensu guide with a way to actually do something -- currently we just end after installing the agent, but this is a crucial point when we need to get users to actually achieve something with Sensu to start building their excitement to use it"

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [ ] new doc (requires a completely new doc)
- [ ] update (add missing or refresh existing content)
- [x] improvement (improve docs content or format)

### Expected behavior
Users should have options for next steps that allow them to see how Sensu does something.

### Current behavior
Users are directed toward next steps that require quite a bit of effort (generating certificates; securing Sensu, etc).

### Corrections needed
- Rework the end so it provides a path for users who just want to try something with Sensu -- adding a cpu check and sending alerts to Slack perhaps? We may also want to update this to use some of the newer Sensu features, like sensuctl dump > sensuctl create.
- Move commercial features to the very end of the Install Sensu page.
- Clarify the current path in Next Steps as just one of the possible options users have now that they have installed Sensu.

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
- Adds a quick fourth step to the end of the agent section to give users another event that is not a keepalive
- Divides the next steps into some purpose-based options
- Adds several pathways that are targeted toward users with less Sensu experience to demonstrate a few interesting and useful things Sensu can do

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/2555

### CODE REVIEW

**Comment by hillaryfraley** [comment]
Maybe the auth guide (https://docs.sensu.io/sensu-go/latest/installation/auth/) should be added as a potential pathway

File: content/sensu-go/5.21/installation/install-sensu.md [hunk 1]
```diff
@@ -416,6 +416,57 @@ For a complete list of configuration options, see the [agent reference][7].
...
+- Manually trigger an event that [sends alerts to your email inbox][12].
+- [Create a check to monitor CPU usage][9] and [send Slack alerts based on your check][10].
+- [Collect metrics][33] with a Sensu check and use a handler to [populate metrics in InfluxDB][37].
+- Use the [sensuctl dump][38] command to export all of your events and resources as a backup &mdash; then use [sensuctl create][39] to restore if needed.
+
+### Deploy Sensu outside your local development environment
+
+If you're going to deploy Sensu for use outside of a local development environment, you should secure it using transport layer security (TLS).
+The next step in setting up TLS is to [generate the certificates you need][2].
+
```
Comment (hillaryfraley) [inline content/sensu-go/5.21/installation/install-sensu.md:461]: Need to add brief explanation of cluster and explain that users should decide at this point whether they want to run a cluster. This should not be separate from the "Deploy outside..." section.

File: content/sensu-go/5.21/installation/install-sensu.md [hunk 2]
```diff
@@ -416,6 +416,57 @@ For a complete list of configuration options, see the [agent reference][7].
...
+If you're ready to see what Sensu can do, one of these pathways can get you started:
+
+- Manually trigger an event that [sends alerts to your email inbox][12].
+- [Create a check to monitor CPU usage][9] and [send Slack alerts based on your check][10].
+- [Collect metrics][33] with a Sensu check and use a handler to [populate metrics in InfluxDB][37].
+- Use the [sensuctl dump][38] command to export all of your events and resources as a backup &mdash; then use [sensuctl create][39] to restore if needed.
+
+### Deploy Sensu outside your local development environment
+
+If you're going to deploy Sensu for use outside of a local development environment, you should secure it using transport layer security (TLS).
```
Comment (hillaryfraley) [inline content/sensu-go/5.21/installation/install-sensu.md]:
```suggestion
To deploy Sensu for use outside of a local development environment, first decide whether you want to [run a Sensu cluster][22].
A Sensu cluster is a group of three or more sensu-backend nodes, each connected to a shared etcd cluster, using Sensu’s embedded etcd or an external etcd cluster.

Clustering allows you to absorb the loss of a backend node, prevent data loss, and distribute the network load of agents.
However, clustered deployments have different configuration implications than single backend deployments, so we recommend deciding whether you will run a Sensu cluster before you generate certificates and set up TLS.

No matter whether you deploy a single backend or a clustered configuration, begin by securing Sensu with transport layer security (TLS).
```

File: content/sensu-go/5.21/installation/install-sensu.md [hunk 3]
```diff
@@ -416,6 +416,57 @@ For a complete list of configuration options, see the [agent reference][7].
...
+The next step in setting up TLS is to [generate the certificates you need][2].
```
Comment (hillaryfraley) [inline content/sensu-go/5.21/installation/install-sensu.md]:
```suggestion

The first step in setting up TLS is to [generate the certificates you need][2].
```

File: content/sensu-go/5.21/installation/install-sensu.md [hunk 4]
```diff
@@ -416,6 +416,57 @@ For a complete list of configuration options, see the [agent reference][7].
...
+
+### Run a Sensu cluster
+
+If you plan to set up a cluster, here's our suggested pathway:
+
+1. [Generate certificates][2].
+2. [Secure Sensu][8].
+3. [Run a Sensu cluster][22].
```
Comment (hillaryfraley) [inline content/sensu-go/5.21/installation/install-sensu.md]:
```suggestion
Then, follow our [Secure Sensu][8] guide to make Sensu production-ready.

After you've secured Sensu, read [Run a Sensu cluster][22] if you are setting up a clustered configuration.
```

File: content/sensu-go/5.21/installation/install-sensu.md [hunk 5]
```diff
@@ -416,6 +416,59 @@ For a complete list of configuration options, see the [agent reference][7].
...
+### Deploy Sensu outside your local development environment
+
+To deploy Sensu for use outside of a local development environment, first decide whether you want to [run a Sensu cluster][22].
+A Sensu cluster is a group of three or more sensu-backend nodes, each connected to a shared etcd cluster, using Sensu’s embedded etcd or an external etcd cluster.
+
+Clustering allows you to absorb the loss of a backend node, prevent data loss, and distribute the network load of agents.
+However, clustered deployments have different configuration implications than single backend deployments, so we recommend deciding whether you will run a Sensu cluster before you generate certificates and set up TLS.
+
+No matter whether you deploy a single backend or a clustered configuration, begin by securing Sensu with transport layer security (TLS).
+
```
Comment (hillaryfraley) [inline content/sensu-go/5.21/installation/install-sensu.md]:
```suggestion
```

**Comment by hillaryfraley** [comment]
@cwjohnston I added the updates re: LDE and clustering as we discussed this afternoon. It's a big improvement in terms of context and laying out a more advanced pathway...thank you for your suggestions. I think this is ready for your review!

edit: I immediately wondered if we might want to mention assets/plugins in one of the "do something new" steps. Maybe change the second pathway option to "...and use a Sensu plugin to send Slack alerts based on your check."

File: content/sensu-go/5.21/installation/install-sensu.md [hunk 6]
```diff
@@ -416,6 +416,58 @@ For a complete list of configuration options, see the [agent reference][7].
...
+- [Collect metrics][33] with a Sensu check and use a handler to [populate metrics in InfluxDB][37].
+- Use the [sensuctl dump][38] command to export all of your events and resources as a backup &mdash; then use [sensuctl create][39] to restore if needed.
+
+### Deploy Sensu outside your local development environment
+
+To deploy Sensu for use outside of a local development environment, first decide whether you want to [run a Sensu cluster][22].
+A Sensu cluster is a group of three or more sensu-backend nodes, each connected to a shared etcd cluster, using Sensu’s embedded etcd or an external etcd cluster.
+
+Clustering allows you to absorb the loss of a backend node, prevent data loss, and distribute the network load of agents.
+However, clustered deployments have different configuration implications than single backend deployments, so we recommend deciding whether you will run a Sensu cluster before you generate certificates and set up TLS.
```
Comment (cwjohnston) [inline content/sensu-go/5.21/installation/install-sensu.md]:
```suggestion
However, scaling a single backend to a cluster, or migrating a cluster from cleartext HTTP to encrypted HTTPS without downtime can require [a number of tedious steps][40].  For this reason, we recommend you decide whether your deployment will require clustering as part of your initial planning effort.
```

File: content/sensu-go/5.21/installation/install-sensu.md [hunk 7]
```diff
@@ -475,6 +519,11 @@ If you plan to set up a cluster, here's our suggested pathway:
...
 [30]: ../../reference/backend#initialization
 [31]: ../../guides/deploying/
+[32]: http://localhost:3000/
+[33]: ../../guides/extract-metrics-with-checks/
 [34]: https://account.sensu.io/
 [35]: ../../api/health/
 [36]: #4-open-the-web-ui
+[37]: ../../guides/influx-db-metric-handler/
+[38]: ../../sensuctl/create-manage-resources/#export-resources
+[39]: ../../sensuctl/create-manage-resources/#create-resources
```
Comment (cwjohnston) [inline content/sensu-go/5.21/installation/install-sensu.md:543]:
```suggestion
[39]: ../../sensuctl/create-manage-resources/#create-resources
[40]: https://etcd.io/docs/v3.3.13/op-guide/runtime-configuration/
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]

File: content/sensu-go/5.21/installation/install-sensu.md [hunk 8]
```diff
@@ -416,6 +416,58 @@ For a complete list of configuration options, see the [agent reference][7].
...
+- [Collect metrics][33] with a Sensu check and use a handler to [populate metrics in InfluxDB][37].
+- Use the [sensuctl dump][38] command to export all of your events and resources as a backup &mdash; then use [sensuctl create][39] to restore if needed.
+
+### Deploy Sensu outside your local development environment
+
+To deploy Sensu for use outside of a local development environment, first decide whether you want to [run a Sensu cluster][22].
+A Sensu cluster is a group of three or more sensu-backend nodes, each connected to a shared etcd cluster, using Sensu’s embedded etcd or an external etcd cluster.
+
+Clustering allows you to absorb the loss of a backend node, prevent data loss, and distribute the network load of agents.
+However, scaling a single backend to a cluster, or migrating a cluster from cleartext HTTP to encrypted HTTPS without downtime can require [a number of tedious steps][40].  For this reason, we recommend you decide whether your deployment will require clustering as part of your initial planning effort.
```
Comment (hillaryfraley) [inline content/sensu-go/5.21/installation/install-sensu.md]:
```suggestion
However, scaling a single backend to a cluster or migrating a cluster from cleartext HTTP to encrypted HTTPS without downtime can require [a number of tedious steps][40].
For this reason, we recommend that you decide whether your deployment will require clustering as part of your initial planning effort.
```

File: content/sensu-go/5.21/installation/install-sensu.md [hunk 9]
```diff
@@ -429,6 +429,59 @@ For a complete list of configuration options, see the [agent reference][7].
...
+
+- Manually trigger an event that [sends alerts to your email inbox][12].
+- [Create a check to monitor CPU usage][9] and [send Slack alerts based on your check][10].
+- [Collect metrics][33] with a Sensu check and use a handler to [populate metrics in InfluxDB][37].
+- Use the [sensuctl dump][38] command to export all of your events and resources as a backup &mdash; then use [sensuctl create][39] to restore if needed.
+
+### Deploy Sensu outside your local development environment
+
+To deploy Sensu for use outside of a local development environment, first decide whether you want to [run a Sensu cluster][22].
+A Sensu cluster is a group of three or more sensu-backend nodes, each connected to a shared etcd cluster, using Sensu’s embedded etcd or an external etcd cluster.
```
Comment (cwjohnston) [inline content/sensu-go/5.21/installation/install-sensu.md:473]:
```suggestion
A Sensu cluster is a group of three or more sensu-backend nodes, each connected to a shared database, provided either by Sensu’s embedded etcd or an external etcd cluster.
```

**Pull request merged**